dlls/nvcuda/nvcuda.c

```diff
@@ -3651,23 +3652,122 @@ CUresult WINAPI wine_cuDeviceGetUuid(CUuuid *uuid, CUdevice dev)
     else return CUDA_ERROR_INVALID_VALUE;
 }
 
+/* code borrowed from Wine starts here */
```

Comment:
Feel free to move this around as you see fit :slightly_smiling_face:

I don't have any preferences myself, I just dumped them here because that was the easiest thing to do :see_no_evil: